src/_help/changes-management/changelog.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Changelog
+---
+
+- TOC
+{:toc}
+
+The changelog allows you to quickly track the breaking changes in an API documentation.
```

Comment:
Nope, it allows you to follow what's happening on your API, or on your API documentation :) (and please remove the "quickly", same as "easily" we shouldn't use this in the doc imo)